source/GameAction.cpp

```diff
@@ -214,6 +215,16 @@ void GameAction::LoadSingle(const DataNode &child, const string &missionName)
 			GameData::Missions().Get(toFail);
 		}
 	}
+	else if(key == "relocate")
+	{
+		for(const DataNode &grand : child)
+		{
+			if(grand.Token(0) == "planet" && grand.Size() > 1)
+				teleportPlanet = GameData::Planets().Get(grand.Token(1));
```

Comment:
The keyword is nicely renamed to "relocate" (which I think is a good improvement), should we do the same for internal keywords; rename the teleport functionality to relocate throughout the codebase?

Reply:
Seems logic, will do this

Reply:
done